explore/src/main/scala/explore/targets/TargetSummaryTable.scala

```diff
@@ -174,34 +184,55 @@ object TargetSummaryTable extends TableHooks:
             acceptLabel = "Yes, delete",
             position = DialogPosition.Top,
             accept = props.targets
-              .mod(_.filter((id, _) => !selectedRowsIds.contains(id))) *> TargetSummaryActions
-              .deleteTargets(selectedRowsIds, props.programId)
-              .runAsyncAndForget,
+              .mod(_.filter((id, _) => !selectedRowsIds.contains(id))) *>
+              table.toggleAllRowsSelected(false) *>
+              TargetSummaryActions
+                .deleteTargets(selectedRowsIds, props.programId)
+                .runAsyncAndForget,
             acceptClass = PrimeStyles.ButtonSmall,
             rejectClass = PrimeStyles.ButtonSmall,
             icon = Icons.SkullCrossBones.color("red")
           )
 
+        def onTextChange(e: ReactEventFromInput): Callback =
+          val files = e.target.files.toList
+          // set value to null so we can reuse the import button
+          (Callback(e.target.value = null) *> filesToImport.set(files)).when_(files.nonEmpty)
```

Comment:
What does the setting to `null` do?

Reply:
Once you select a file value is set, thus you cannot click again to open another one. The solution is to set it to null thus allowing it to be set again

Reply:
Thanks for the information. I never would have guessed.

Reply:
Me neither, it took me a while and somo stack overflow searches to find out